build.sbt

```diff
@@ -5525,6 +5526,27 @@ lazy val `std-table` = project
   .dependsOn(`poi-wrapper`)
   .dependsOn(`std-base` % "provided")
 
+lazy val `std-tests` = project
```

Comment:
@jdunkerley, it is possible to run these tests quickly:
```
sbt:enso> std-tests/clean
sbt:enso> std-tests/test
[info] Test run org.enso.base.polyglot.tests.EnsoMetaTest finished: 1 total, 3.595s
[info] Passed: Total 1, Failed 0, Errors 0, Passed 1
```
e.g. the test finished in less than **four seconds**.